src/java/org/apache/cassandra/db/transform/FilteredPartitions.java

```diff
@@ -25,7 +25,6 @@
 import org.apache.cassandra.db.partitions.UnfilteredPartitionIterator;
 import org.apache.cassandra.db.rows.RowIterator;
 
-// TODO (review): Why was this final?
 public class FilteredPartitions extends BasePartitions<RowIterator, BasePartitionIterator<?>> implements PartitionIterator
```

Comment:
as far as i could tell this could still be `final`, but since the TODO above wasn't answered in the review, i just removed it to simplify the diff